CRM/Utils/JSON.php

```diff
@@ -39,16 +39,28 @@
 class CRM_Utils_JSON {
 
   /**
-   * Function to create JSON object
+   * Output json to the client.
+   * @param mixed $input
+   */
+  public static function output($input) {
+    header('Content-Type: application/json');
+    CRM_Utils_JSON::output($input);
```

Comment:
Hmm, that's not the code I see in master...

On 02/18/2015 03:24 PM, Eileen McNaughton wrote:
>
> In CRM/Utils/JSON.php 
> <https://github.com/civicrm/civicrm-core/pull/5173#discussion_r24937122>:
>
> > @@ -39,16 +39,28 @@
> >  class CRM_Utils_JSON {
> >
> >    /**
> > -   * Function to create JSON object
> > +   * Output json to the client.
> > +   * @param mixed $input
> > +   */
> > +  public static function output($input) {
> > +    header('Content-Type: application/json');
> > +    CRM_Utils_JSON::output($input);
>
> dang. I just copied the function from master - maybe I got your first 
> version of it...
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/civicrm/civicrm-core/pull/5173/files#r24937122>.
>